kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -16,6 +16,19 @@
 # You should have received a copy of the GNU General Public License
 # along with kikuchipy. If not, see <http://www.gnu.org/licenses/>.
 
+
+from typing import Optional, Union
+
+import dask.array as da
+import numpy as np
+from orix.vector import Vector3d
+from orix.quaternion import Rotation
+
+from kikuchipy.detectors.ebsd_detector import EBSDDetector
```

Comment:
The imports should be structured as specified in the contributing guide (https://kikuchipy.org/en/latest/contributing.html#code-style), i.e. the kikuchipy imports below the third party imports (hyperspy, orix).